modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -12832,6 +12892,146 @@ paths:
             tags: ["Sku"]
+    /skus/by-externalReferenceCode/{externalReferenceCode}/subscriptionConfiguration:
+        get:
+            operationId: getSkuByExternalReferenceCodeSubscriptionConfiguration
+            parameters:
+                - in: path
+                  explode: false
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      description:
+                          The resource external reference code
+                      example: AB-34098-789-N
+                      minLength: 1
+                      type: string
+                  style: simple
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/SkuSubscriptionConfiguration"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/SkuSubscriptionConfiguration"
+                    description:
+                        Successful operation
+                400:
+                    description:
+                        Invalid input
+                401:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Authentication information is missing or invalid
+                404:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        The specified resource was not found
+                500:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Unexpected error
+            security:
+                - BasicAuth: []
+                - BearerAuth: []
+                - OAuth2:
+                      - HeadlessCommerceAdminCatalog.read
+            summary: Gets a sku subscription configuration of a product.
+            tags: ["SkuSubscriptionConfiguration"]
+        patch:
+            operationId: patchSkuByExternalReferenceCodeSubscriptionConfiguration
```

Comment:
I saw that this is also in the product api, but I believe this is a mistake also in product API, the pattern for the PATCH is /entities/id, not this one. Moreover this is not a separate entity in the end, this is a DTO that we created to semplify the model, but these fields belong to the SKU, so I think you should move this update model in the SKU PATCH method. For this same reason I think you should also support the creation of this Configuration along with the creation of the SKU so that it would be possible to create everything with just one API call